src/main/kotlin/Decorator/littlekai/ingredients/Chicken.kt

```diff
@@ -4,6 +4,6 @@ import Decorator.littlekai.base.IngredientDecorator
 import Decorator.littlekai.base.Noodles
 
 
-class Chicken(noodles: Noodles) : IngredientDecorator(noodles) {
-  override val COST: Double = 3.50
+class Chicken(noodles: Noodles, val COST: Double = 3.50): IngredientDecorator(noodles) {
```

Comment:
I've done some tests and you're right. I have extracted the COST variable from the constructor to an immutable constant.